project.clj

```diff
@@ -57,7 +57,7 @@
                  ;; Database connectivity
                  [com.zaxxer/HikariCP "2.4.3"]
                  [org.clojure/java.jdbc "0.4.2"]
-                 [org.postgresql/postgresql "9.4-1206-jdbc4"]
+                 [org.postgresql/postgresql "9.4.1208.jre7"]
```

Comment:
There was a dependency conflict with the activity service so I thought I'd just bump it since it's an update anyways.